staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -397,6 +410,13 @@ type Memory struct {
 	// Defaults to the requested memory in the resources section if not specified.
 	// + optional
 	Guest *resource.Quantity `json:"guest,omitempty"`
+	// SharePages instructs the hypervisor whether to disable shared pages (memory merge, KSM) for this domain.
+	// + optional
+	SharePages *bool `json:"sharepages,omitempty"`
+	// LockGuest specifies whether to lock the memory pages belonging to the domain in host's memory,
+	// so that the host is not be allowed to swap them out.
+	// + optional
+	LockGuest *bool `json:"lockguest,omitempty"`
```

Comment:
same here, call the json value `lockGuest`

Reply:
Will do.